util/misc.go

```diff
@@ -513,12 +513,29 @@ func LoadTLSCertificates(ca, key, cert string, autoTLS bool, rsaKeySize int) (tl
 			}
 		}
 	}
+
+	// This excludes ciphers listed in tls.InsecureCipherSuites() and can be used to filter out more
+	var cipherSuites []uint16
+	var cipherNames []string
+	for _, sc := range tls.CipherSuites() {
+		switch cipherName := sc.Name; cipherName {
+		case "TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA", "TLS_RSA_WITH_3DES_EDE_CBC_SHA":
+			logutil.BgLogger().Info("Disabling weak cipherSuite", zap.String("cipherSuite", cipherName))
+		default:
+			cipherNames = append(cipherNames, sc.Name)
```

Comment:
use `cipherName` here? or just don't introduce `cipherName := sc.Name` on line 521?

Reply:
refactored this a bit, PTAL